/usr/ports/pobj/stellarium-23.3/stellarium-23.3/src/StelLogger.cpp:218:49: error: no member named 'SkipEmptyParts' in 'QString'; did you mean 'Qt::SkipEmptyParts'?
        QStringList dmesgLines = dmesgData.split('\n', QString::SkipEmptyParts);
                                                       ^~~~~~~~~~~~~~~~~~~~~~~
                                                       Qt::SkipEmptyParts
/usr/local/include/X11/qt6/QtCore/qnamespace.h:127:9: note: 'Qt::SkipEmptyParts' declared here
        SkipEmptyParts = 0x1,

Index: src/StelLogger.cpp
--- src/StelLogger.cpp.orig
+++ src/StelLogger.cpp
@@ -96,11 +96,7 @@ void StelLogger::init(const QString& logFilePath)
 	lspci.start("lspci", { "-v" }, QIODevice::ReadOnly);
 	lspci.waitForFinished(300);
 	const QString pciData(lspci.readAll());
-	#if (QT_VERSION>=QT_VERSION_CHECK(5, 14, 0))
 	QStringList pciLines = pciData.split('\n', Qt::SkipEmptyParts);
-	#else
-	QStringList pciLines = pciData.split('\n', QString::SkipEmptyParts);
-	#endif
 	for (int i = 0; i<pciLines.size(); i++)
 	{
 		if(pciLines.at(i).contains("VGA compatible controller"))
@@ -180,11 +176,7 @@ void StelLogger::init(const QString& logFilePath)
 	systemProfiler.waitForStarted();
 	systemProfiler.waitForFinished();
 	const QString systemData(systemProfiler.readAllStandardOutput());
-	#if (QT_VERSION>=QT_VERSION_CHECK(5, 14, 0))
 	QStringList systemLines = systemData.split('\n', Qt::SkipEmptyParts);
-	#else
-	QStringList systemLines = systemData.split('\n', QString::SkipEmptyParts);
-	#endif
 	for (int i = 0; i<systemLines.size(); i++)
 	{
 		// hardware overview
@@ -214,11 +206,7 @@ void StelLogger::init(const QString& logFilePath)
 	dmesg.waitForStarted();
 	dmesg.waitForFinished();
 	const QString dmesgData(dmesg.readAll());
-	#if (QT_VERSION>=QT_VERSION_CHECK(5, 14, 0))
-	QStringList dmesgLines = dmesgData.split('\n', QString::SkipEmptyParts);
-	#else
 	QStringList dmesgLines = dmesgData.split('\n', Qt::SkipEmptyParts);
-	#endif
 	for (int i = 0; i<dmesgLines.size(); i++)
 	{
 		if (dmesgLines.at(i).contains("memory"))
